docs/install/README.md

```diff
@@ -371,38 +373,460 @@ Status:
         Updated Replicas:      1
 Events:  <none>
 ```
+## Helm
+You need to have [Helm v3+](https://helm.sh/docs/intro/install/) installed.
+
+Configure the K8ssandra Helm repository:
+
+```console
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
+```
+
+Update your Helm repository cache:
+
+```console
+helm repo update
+```
+
+Verify that you see the `k8ssandra-operator` chart:
+
+```console
+helm search repo k8ssandra-operator
+NAME                               	CHART VERSION	APP VERSION	DESCRIPTION
+k8ssandra/k8ssandra-operator       	0.31.0       	1.0.0      	Kubernetes operator which handles the provision...
+```
 
-## Single Cluster Install with Kustomize
-We will first look at a single cluster install to demonstrate that while K8ssandra Operator is designed for multi-cluster use, it can be used in a single cluster without any extra configuration.
+### Single Cluster 
+We will first look at a single cluster install to demonstrate that while K8ssandra 
+Operator is designed for multi-cluster use, it can be used in a single cluster without 
+any extra configuration.
 
-### Create kind cluster
+#### Create kind cluster
 Run `setup-kind-multicluster.sh` as follows:
 
 ```sh
 ./setup-kind-multicluster.sh --kind-worker-nodes 4
 ```
 
-### Install Cert Manager
+#### Install K8ssandra Operator
+Install the Helm chart:
+
+```console
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator --create-namespace
+```
+This `helm install` command does the following:
+
+* Create the `k8ssandra-operator` namespace if necessary
+* Install Cass Operator in the `k8ssandra-operator` namespace
+* Install K8ssandra Operator in the `k8ssandra-operator` namespace
+
+This does not currently install Cert Manager. Cass Operator requires Cert Manager when 
```

Comment:
cert-manager is only required when the webhook is enabled. I created https://github.com/k8ssandra/k8ssandra/issues/1117 to update the cass-operator chart so that deploys cert-manager.